service/service.go

```diff
@@ -0,0 +1,79 @@
+package service
+
+import (
+	"context"
+	"github.com/ONSdigital/dp-frontend-renderer/assets"
+	"github.com/ONSdigital/dp-frontend-renderer/config"
+	renderHelpers "github.com/ONSdigital/dp-frontend-renderer/render"
+	"github.com/ONSdigital/dp-frontend-renderer/routes"
+	"github.com/ONSdigital/go-ns/handlers/requestID"
+	"github.com/ONSdigital/go-ns/handlers/timeout"
+	deprecatedLog "github.com/ONSdigital/go-ns/log"
+	"github.com/ONSdigital/go-ns/render"
+	"github.com/ONSdigital/log.go/log"
+	"github.com/gorilla/pat"
+	"github.com/justinas/alice"
+	unrolled "github.com/unrolled/render"
+	"html/template"
+	"net/http"
+	"time"
+)
+
+func Run(ctx context.Context, taxonomyRedirects map[string]string) error {
+	log.Namespace = "dp-frontend-renderer"
+
+	cfg, err := config.Get()
+	if err != nil {
+		log.Event(ctx, "unable to retrieve service configuration", log.Error(err))
```

Comment:
I don't think its necessary to log this error here as it you are returning it to the caller. This will be logged by main/Run so logging it here is just duplicating that info. Generally speaking you should either handle an error completely or do nothing and return it to the caller - it helps keep the code cleaner and reduced redundant logging.

If you think it will be useful you can use https://godoc.org/github.com/pkg/errors#Wrap to add additional context to an error:
```
if err != nil {
    return errors.Wrap(err, "unable to retrieve service configuration")
}
```